pkg/proc/linutil/regs_riscv64_arch.go

```diff
@@ -0,0 +1,386 @@
+package linutil
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/go-delve/delve/pkg/dwarf/op"
+	"github.com/go-delve/delve/pkg/dwarf/regnum"
+	"github.com/go-delve/delve/pkg/proc"
+	"golang.org/x/arch/riscv64/riscv64asm"
+)
+
+// Regs is wrapper for sys.PtraceRegs
```

Comment:
The comment needs to start with "RISCV64Registers is..."

Reply:
Fixed.